tests/sqlAst/data/sql-ast-narrowing.php

```diff
@@ -214,28 +214,53 @@ public function strcase(PDO $pdo): void
         assertType('PDOStatement<array{col: non-empty-string&numeric-string, 0: non-empty-string&numeric-string}>', $stmt);
     }
 
-    public function avgMinMax(PDO $pdo): void
+    public function avg(PDO $pdo): void
     {
         $stmt = $pdo->query('SELECT avg(freigabe1u1) as avg from ada');
-        assertType('PDOStatement<array{avg: int<-32768, 32767>|null, 0: int<-32768, 32767>|null}>', $stmt);
+        assertType('PDOStatement<array{avg: numeric-string|null, 0: numeric-string|null}>', $stmt);
```

Comment:
Is a int really turned into a numeric-string?

Or should it be `int<...>|float` instead?

Reply:
For whatever reason, it really is turned into a `numeric-string`.

```php
$result = $pdo->query('SELECT avg(3)')->fetch();
var_dump($result);
$result = $pdo->query('SELECT avg(account_id) from account')->fetch();
var_dump($result);
```
```
array (size=2)
  'avg(3)' => string '3.0000' (length=6)
  0 => string '3.0000' (length=6)
array (size=2)
  'avg(account_id)' => string '23.0256' (length=7)
  0 => string '23.0256' (length=7)
```